src/osin_storages/memory.go

```diff
@@ -0,0 +1,135 @@
+package osin_storages
```

Comment:
So we have to implement a new one? Is there a builtin storage? 

Also , please avoid using underscore in the package name

Reply:
It is copied from osin/example https://github.com/RangelReale/osin/blob/master/example/teststorage.go

A mongo version already exist https://github.com/linkernetworks/oauth/blob/develop/src/app/storage/storage.go

I think in-memory version is more convenient.